airflow/providers/mysql/operators/mysql.py

```diff
@@ -46,6 +46,7 @@ class MySqlOperator(BaseOperator):
     """
 
     template_fields = ('sql',)
+    template_fields_renderers = {'sql': 'sql'}
     template_ext = ('.sql',)
     ui_color = '#ededed'
 
```

Comment:
Should we add `json` renderer for `parameters`?

Reply:
That's a good idea. Lets' do it!